src/pybind/mgr/dashboard/controllers/feedback.py

```diff
@@ -0,0 +1,34 @@
+# # -*- coding: utf-8 -*-
+from typing import Optional
+
+from .. import mgr
+from ..services import feedback
+from . import ApiController, ControllerDoc, RESTController
+from ..model.feedback import Feedback
+from ..security import Scope
+
+@ApiController('/feedback', Scope.CONFIG_OPT)
+@ControllerDoc("Feedback API", "Report")
+class FeedbackController(RESTController):
+    issueAPIkey = None
+
+    def __init__(self):  # pragma: no cover
+        super(FeedbackController, self).__init__()
+        self.cephTrackerClient = feedback.CephTrackerClient()
+
+    def create(self, project_id, tracker_id, subject, description, severity: Optional[str] = '3 - Minor'):
+        """
+        Create an issue.
+        :param component: The buggy ceph component.
+        :param title: The title of the issue.
+        :param description: The description of the issue.
+        """
+        feedback = Feedback(project_id, tracker_id, subject, description, severity)
+        return self.cephTrackerClient.create_issue(feedback)
+
+    def get(self, issue_number):
+        """
+        Validate the issue tracker API given by user.
+        :param issueAPI: The issue tracker API access key.
+        """
+        return self.cephTrackerClient.get_issues(issue_number)
```

Comment:
```suggestion
        try:
            return self.cephTrackerClient.get_issues(issue_number)
        except RequestException as error:
            if error.status_code == 404:
                raise DashboardException(msg=f'Issue {issue_number} not found',
                                         http_status_code=error.status_code,
                                         component='feedback')
            raise error
```

Return 404 for non-existing issues.